diff -up qtscriptgenerator-src-0.1.0/qtbindings/qtbindings.pro.no_phonon qtscriptgenerator-src-0.1.0/qtbindings/qtbindings.pro
--- qtscriptgenerator-src-0.1.0/qtbindings/qtbindings.pro.no_phonon	2009-02-20 05:42:24.000000000 -0600
+++ qtscriptgenerator-src-0.1.0/qtbindings/qtbindings.pro	2009-03-10 09:54:04.000000000 -0500
@@ -6,7 +6,6 @@ SUBDIRS = qtscript_core \
 	  qtscript_sql \
 	  qtscript_svg \
 	  qtscript_xml \
-	  qtscript_phonon \
 	  qtscript_webkit \
 	  qtscript_xmlpatterns \
 	  qtscript_uitools \
